packages/pdf-worker/src/templates/ChatTranscript/markup/blocks/UnorderedListBlock.tsx

```diff
@@ -0,0 +1,32 @@
+import { StyleSheet, Text, View } from '@react-pdf/renderer';
+import type * as MessageParser from '@rocket.chat/message-parser';
+
+import InlineElements from '../elements/InlineElements';
+
+const styles = StyleSheet.create({
+	wrapper: {
+		marginTop: 4,
+	},
+	list: {
+		flexDirection: 'row',
+	},
```

Comment:
Maybe you should also define `display: flex` here 🤔 . 
I know the `View` component probably already has it, but I don't trust the libraries to not change.
I'll leave to your judgement.